package.json

```diff
@@ -35,6 +35,13 @@
       "pre-commit": "lint-staged"
     }
   },
+  "resolutions": {
+    "@jupyterlite/contents": "file:/home/trungle/WORK/jupyterlite/packages/contents",
```

Comment:
FYI it should now be possible to build against released packages: https://www.npmjs.com/package/@jupyterlite/contents/v/0.2.0-alpha.0

https://github.com/jupyterlite/jupyterlite/releases/tag/v0.2.0a0